tomviz/vtkCustomPiecewiseControlPointsItem.cxx

```diff
@@ -0,0 +1,52 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+#include "vtkCustomPiecewiseControlPointsItem.h"
+
+#include "vtkContextMouseEvent.h"
+#include "vtkContextScene.h"
+#include "vtkObjectFactory.h"
+
+//-----------------------------------------------------------------------------
+vtkStandardNewMacro(vtkCustomPiecewiseControlPointsItem);
```

Comment:
Carry over from VTK style, which must have suppressions for these warnings? I haven't seen them, at least, and almost every class implementation file has a semi-colon after the usual VTK macros.

In general, do we want to follow VTK style for VTK-derived classes? Or tomviz style?